resources/config.rb

```diff
@@ -16,7 +16,7 @@ def content
     next n.flatten.map(&:to_s).join(',') if n.is_a?(Array)
     n
   end.map do |k, v|
-    'GRUB_' << k.upcase << (v.is_a?(String) ? "=\"#{v}\"" : "=#{v}")
+    'GRUB_' << k.upcase << (v =~ /[\W\s]+/ ? "=\"#{v}\"" : "=#{v}")
```

Comment:
Should this clause be `v.is_a?(String) && v =~ /[\W\s]+/` ?

Reply:
Note:
```
irb(main):005:0> v = 'has a space'.to_sym
=> :"has a space"
irb(main):006:0> v =~ /[\W\s]+/
=> 3
```

Reply:
thanks @jjustice6 updated

Reply:
This LGTM now